self-tests.yml

```diff
@@ -208,6 +208,40 @@ functions:
             make mod-tidy;
             [[ $? -eq 0 ]] && break;
           done
+  
+  generate-api-docs:
+    - command: subprocess.exec
+      params:
+        binary: make
+        args: ["swaggo-install"]
+        working_dir: evergreen
+        env:
+          GOPATH: ${workdir}/gopath
+        include_expansions_in_env:
+          - GOROOT
+    - command: shell.exec
+      params:
+        shell: bash
+        working_dir: evergreen
+        env:
+          GOPATH: ${workdir}/gopath
+        script: |
+          set -o verbose
+          set -o errexit
+          $GOPATH/bin/swag init -g service/service.go
+  
+  push-api-docs-task:
```

Comment:
nit: would be can we call this something else since it's a function and not a task?

Reply:
Sure, I combined to just use the `generate-api-docs` task since the tasks are essentially the same. The `s3.put` calls have `patchable` to `false`.

Reply:
(also removed the functions)